python/cuml/test/test_fil.py

```diff
@@ -302,7 +302,8 @@ def test_fil_skl_regression(n_rows, n_columns, n_estimators, max_depth,
     fil_preds = np.asarray(fm.predict(X_validation))
     fil_mse = mean_squared_error(y_validation, fil_preds)
 
-    assert fil_mse == pytest.approx(skl_mse, 1e-4)
```

Comment:
Does this expression cause a failure if FIL is better than SKL?

Reply:
yes. one might argue that we need to expand the approximation interval, since there's no way reordering would only make MSE better by an amount.